README.md

```
@@ -67,8 +67,19 @@ to test:
 `npx wrangler d1 execute production --local --command="SELECT * FROM users"`
 
 For local development add SSL certificate(you will get issue in the browser regarding insecure connection):
-`sudo security add-trusted-cert -d -r trustRoot -k "/Library/Keychains/System.keychain" certificates/localhost.pem`
-Certificate should be generated after in `./certificates` directory after first run on `npm run dev` - wrangler dev with https flag
+`sudo security add-trusted-cert -d -r trustRoot -k "/Library/Keychains/System.keychain" certificates/localhost.crt`
+
+To generate certificate on MacOS run
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)